src/test/kotlin/io/github/mojira/arisa/modules/AttachmentModuleTest.kt

```diff
@@ -35,7 +39,9 @@ class AttachmentModuleTest : StringSpec({
 
     "should return FailedModuleResponse when deleting fails" {
         val module = AttachmentModule(listOf(".test"))
-        val attachment = Attachment("testfile.test", NOW, { RuntimeException().left() }, { ByteArray(0) })
+        val attachment = getAttachment(
+            remove = { RuntimeException().left() }
+        )
```

Comment:
IIRC this can be `getAttachment() { RuntimeException().left() }`, because it's the last parameter and it's also a function

Reply:
I suppose writing it as such can slightly improve readability and consistency with some other 
crazy domain classes like `Comment` which has two parameters (`restrict` and `update` ) of type function

Reply:
I prefer making it explicit